### ISSUE TITLE
create string freeze process

### ISSUE THREAD

**Issue by aceman444** [osmandapp/Osmand#3417]
Hi.

I translate OsmAnd strings almost daily. But it happened that somehow the release of OsmAnd 2.5 still contained some untranslated strings. I translated those soon, but they are still not included even in 2.5.4.

This should not happen as it greatly reduces the apparent quality of OsmAnd to end-users.

I propose to create a process that allows translators (localizers) to finish all strings in time for a release:
1.There should be a period of time before a release of new version when the developers promise to not change/add new strings into the application. This is the string freeze.
2.Then while point releases (2.5.x) are still produced, the translated strings should still be merged into the release (e.g. if somebody translates a string between release of 2.5.2 and 2.5.3, the translation should get into the release.

For 1, the string freeze time should be appropriate for OsmAnd mode of development (I do not ask for 3 *months* which e.g. projects based on Firefox have, but then they have multiple development branches in parallel). Maybe something like 2 weeks. The important part is the imminent release is loudly announced on the OsmAnd mailing list.
For 2, if OsmAnd only has one development branch, then it would probably need to avoid development while important fixes are still pushed into the point releases. So that the strings (and translations) are not branched away from the release branch (which seems to be the case now).

Thanks.

**Comment by vshcherb** [comment]
Agree, though we had some cases in the past where last translated strings crashed the app. So, we usually branch out before 1 week of public release.

In that sense we need to freeze both development and translation. Though your point is right we need to announce it and keep development frozen in sense of creating new strings for 1 week at least before we create a branch. 

I will keep in mind
BTW next release is likely to be in mid Februrary.

**Comment by aceman444** [comment]
OK, that's fine, we just need the announcement on the mailing list, when the freeze starts and when it ends (when you branch before release). Thanks

**Comment by aceman444** [comment]
There seems to be beta of OsmAnd 3.3 nearing and we got no announcement on mailinglist yet.

**Comment by vshcherb** [comment]
OsmAnd 3.3 will take at least 2 more weeks, I'm afraid
